Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "linked_list_allocator"
-version = "0.8.3"
+version = "0.8.4"
```

Comment:
I normally do the version bumping as part of a separate publish commit, so could you please remove the version bump from this PR?

Reply:
Sure. (Thought to make it easy, but it makes sense.)